workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/metrics.py

```diff
@@ -118,24 +163,30 @@ def time_and_global_mean_value(diags):
     return time_and_global_mean_value
 
 
-@add_to_metrics("time_and_global_mean_bias")
-def time_and_global_mean_bias(diags):
-    time_mean_bias = grab_diag(diags, "time_mean_bias")
-    area = diags["area"]
-    time_and_global_mean_bias = weighted_mean(time_mean_bias, area, HORIZONTAL_DIMS)
-    restore_units(time_mean_bias, time_and_global_mean_bias)
-    return time_and_global_mean_bias
+for mask_type in ["global", "land", "sea"]:
```

Comment:
Could you add the `land` and `sea` metrics for the `time_and_global_mean_value` above? Nice to have the absolute value in addition to the bias.